numpy/f2py/crackfortran.py

```diff
@@ -2553,8 +2552,7 @@ def get_parameters(vars, global_params={}):
             elif iscomplex(vars[n]):
                 # FIXME complex numbers may also have exponents
                 if v[0] == '(' and v[-1] == ')':
-                    # FIXME, unused l looks like potential bug
-                    l = markoutercomma(v[1:-1]).split('@,@')
+                    pass
```

Comment:
Does this reflect a deeper problem? Should the result of `markeroutercomma` be used somewhere?

I think this should be left for a future clean-up PR.

Reply:
This looks like an unfinished implementation. In short term, replace the block under `elif iscomplex(...):` with
```
outmess(f'get_parameters[TODO]: implement evaluation of complex expression {v}')
```

Reply:
Looks like this is resolved.